src/App.tsx

```diff
@@ -151,6 +153,8 @@ const AppRoutes = ({ base }: AppProps) => {
                     <Route index element={<FoldersPage />} />
                     <Route path=":folderId" element={<FoldersPage />} />
                   </Route>
+                  <Route path="arena" element={<ArenaPage />} />
+                  <Route path="arena/:topic" element={<TopicPage />} />
```

Comment:
her burde du heller ta imot en kategori-id 
```suggestion
                  <Route path="arena/:categoryId" element={<TopicPage />} />
```